test/toy_module_test.rb

```diff
@@ -1,10 +1,11 @@
 require 'test_helper'
+require 'toy_class'
```

Comment:
Right, I guess we forgot this all the way back in 55d19a372d3cd620e52b1edeaf032fe27516a37b.